### DISCUSSION TITLE
Documentation Change

### DISCUSSION THREAD

**Discussion by RaZ0rr-Two** [davatorium/rofi#1494] [General]
Can this [link](https://github.com/davatorium/rofi/wiki/Configuring-Rofi)  be moved to a better spot? I think it has some useful information. So, putting it in wiki with 'currently unmainted' besides it might turn users away. I think 'configuration' and 'installation' can be given their own pages and rest in Wiki. But to be honest I think the 'Dcoumentation' section in [this](https://github.com/davatorium/rofi/wiki) link should be given as 'Quickstart' section in readme.

**Comment by DaveDavenport** [comment]
Not possible, unless I get help with documentation (see https://github.com/davatorium/rofi/issues/1019,  https://github.com/davatorium/rofi/discussions/1432, https://www.reddit.com/r/qtools/comments/jqdv1y/support_with_themes_and_documentation/, https://github.com/davatorium/rofi/issues/861, etc.).

**Reply by RaZ0rr-Two** [reply]
Yes, I agree that maintaining the theme section is difficult but [this](https://github.com/davatorium/rofi/wiki/Configuring-Rofi) is absolutely okay for the current version. Do you think it can be moved to the readme?

**Reply by DaveDavenport** [reply]
Readme seems to be the wrong place,  manpages seems to be more suitable.
Most of the page is copied from the manpage...

**Reply by DaveDavenport** [reply]
In short, I have no intention to spend  effort on documentation beside updating the manpage.
If somebody wants to help, this would be highly appreciated.

**Reply by RaZ0rr-Two** [reply]
Ok, I will try

**Reply by DaveDavenport** [reply]
Thanks! :partying_face:

**Comment by RaZ0rr-Two** [comment]
Hello Dave,

Could you look at my [repo](https://github.com/RaZ0rr-Two/rofi)? I have organised everything. I will make more changes after hearing your feedback.

**Reply by DaveDavenport** [reply]
https://github.com/davatorium/rofi/pull/1495 turned it in a pull request so its easier to review.

**Reply by DaveDavenport** [reply]
added some notes, will look at it more tomorrow

**Reply by RaZ0rr-Two** [reply]
Ok. My main aim here is to organise all the available information. 'themes' are one of my weak departments as I simply copy the themes rasi file from other repos. I have added some such repo in the themes section in wiki under 'examples'. So, I need your help there. However, I believe the document has become beginner-friendly and new users can jump into it more quickly. All there's left to do is reviewing the content by experts like yourself

**Comment by RaZ0rr-Two** [comment]
Dave, have you looked at it? whats the next step?

**Comment by RaZ0rr-Two** [comment]
Have made the icons to releases and discussions in the centre
